src/app/user/user-form.component.ts

```diff
@@ -79,15 +84,21 @@ export class UserFormComponent implements OnInit {
   }
 
   toggleNotifyAppointments() {
-    if (!this.pushSubscription || !this.model.notifications.appointment) {
+    if (!this.pushSubscription || !this.model || !this.model.notifications) {
       return;
     }
 
+    if (!this.model.notifications.appointment) {
```

Comment:
`this.model.notifications.appointment` should contain a list of object id's. Those always get lost on the client side on my tests. The db and the response of the request contain them, but they're not in `this.model`. Maybe it's because they're of the type `ObjectId` and the parsing from json can't handle them??